src/core/dbus-execute.c

```diff
@@ -1324,6 +1348,9 @@ const sd_bus_vtable bus_exec_vtable[] = {
         SD_BUS_PROPERTY("ProtectHostname", "b", bus_property_get_bool, offsetof(ExecContext, protect_hostname), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("NetworkNamespacePath", "s", NULL, offsetof(ExecContext, network_namespace_path), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("IPCNamespacePath", "s", NULL, offsetof(ExecContext, ipc_namespace_path), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("RootImagePolicy", "s", property_get_image_policy, offsetof(ExecContext, root_image_policy), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("MountImagePolicy", "s", property_get_image_policy, offsetof(ExecContext, mount_image_policy), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
so there's only one RootImage, so for that case it's fine. But there can be multiple extensions/images, shouldn't there be the possibility of setting a different policy per image?

Reply:
I mean, sure, one could do it that way, but I am not convinced of the usecase too much for now, and if there's really a super-strong usecase for it we can revisit this. It would blow up the complexity, and security and complexity are forces pushing in different directions.

The way I see it security/access policies are best associated to *types* of objects instead of objects themselves. This is what things like selinux put a lot of focus on. But of course, this really depends on the usecase.

Moreover I think if we truly wanted to allow defining per-image policies we probably should put them in sidecar files or xattrs, so that they are truly close to the image, and not to external configuration. i.e. I see a logic like this:

1. per-object policy: encode policy object
2. per-type policy: encode policy as part of external concept defining/using objects

Reply:
Then make this a single policy? `DefaultImagePolicy` or so? I'm not sure there's that much of a difference between these to say they are different object types. Then in the future more granular ones can be added if needed

Reply:
Then make this a single policy? `DefaultImagePolicy` or so? I'm not sure there's that much of a difference between these to say they are different object types. Then in the future more granular ones can be added if needed

Reply:
So we definitely should have DefaultRootImagePolicy=, DefaultMountImagePolicy=, DefaultExtensionImagePolicy=. Just didn't want to add even more commits to this PR, as mentioned earlier.

The reason why I added this primarily as per-unit settings for now is that I actually want that for portable services the default should be inherited from `portabled.conf` I think. 

But then I shied away from actually adding this to system.conf, portabled.conf, sysext.conf just yet, because this PR is already too large.

But I think we should keep the three `…RootImage…=`, `…MountImage…=` and `…ExtensionImage…=` policies separate on all levels, since they are used quite differently. i.e. for the root image it might make sense to mount /srv of the DDI, but for extension image definitely not. And for mount image stuff you might want to have the most restrictive ddi. That's what I mean by "types", i.e. keep the policies for the *types* separate